### PR TITLE
add stable aliases for deleted docs in dev

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5264
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: loader-overview.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Loader 使用文档
 summary: 了解 Loader 工具以及使用方法。
-aliases: ['/docs-cn/stable/loader-overview/','/docs-cn/v4.0/loader-overview/','/docs-cn/stable/reference/tools/loader/','/docs-cn/stable/load-misuse-handling/','/docs-cn/v4.0/reference/tools/loader','/docs-cn/v4.0/reference/tools/error-case-handling/load-misuse-handling/']
+aliases: ['/docs-cn/stable/loader-overview/','/docs-cn/v4.0/loader-overview/','/docs-cn/stable/reference/tools/loader/','/docs-cn/stable/load-misuse-handling/','/docs-cn/v4.0/reference/tools/loader','/docs-cn/v4.0/reference/tools/error-case-handling/load-misuse-handling/','/zh/tidb/stable/loader-overview','/zh/tidb/dev/loader-overview']
```
Comment (TomShawn) [inline loader-overview.md:4]:
```suggestion
aliases: ['/docs-cn/stable/loader-overview/','/docs-cn/v4.0/loader-overview/','/docs-cn/stable/reference/tools/loader/','/docs-cn/stable/load-misuse-handling/','/docs-cn/v4.0/reference/tools/loader','/docs-cn/v4.0/reference/tools/error-case-handling/load-misuse-handling/','/zh/tidb/stable/loader-overview']
```

File: mydumper-overview.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Mydumper 使用文档
 summary: 使用 Mydumper 从 TiDB 导出数据。
-aliases: ['/docs-cn/stable/mydumper-overview/','/docs-cn/v4.0/mydumper-overview/','/docs-cn/stable/reference/tools/mydumper/']
+aliases: ['/docs-cn/stable/mydumper-overview/','/docs-cn/v4.0/mydumper-overview/','/docs-cn/stable/reference/tools/mydumper/','/zh/tidb/stable/mydumper-overview','/zh/tidb/dev/mydumper-overview']
```
Comment (TomShawn) [inline mydumper-overview.md:4]:
```suggestion
aliases: ['/docs-cn/stable/mydumper-overview/','/docs-cn/v4.0/mydumper-overview/','/docs-cn/stable/reference/tools/mydumper/','/zh/tidb/stable/mydumper-overview']
```

File: experimental-features-4.0.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: TiDB 4.0 实验特性
 summary: 了解 TiDB 4.0 版本的实验特性。
-aliases: ['/docs-cn/stable/experimental-features-4.0/','/docs-cn/v4.0/experimental-features-4.0/']
+aliases: ['/docs-cn/stable/experimental-features-4.0/','/docs-cn/v4.0/experimental-features-4.0/','/tidb/stable/experimental-features-4.0']
```
Comment (yikeke) [inline experimental-features-4.0.md:4]:
```suggestion
aliases: ['/docs-cn/stable/experimental-features-4.0/','/docs-cn/v4.0/experimental-features-4.0/','/zh/tidb/stable/experimental-features-4.0']
```

File: syncer-overview.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Syncer 使用文档
-aliases: ['/docs-cn/stable/syncer-overview/','/docs-cn/v4.0/syncer-overview/','/docs-cn/stable/reference/tools/syncer/']
+aliases: ['/docs-cn/stable/syncer-overview/','/docs-cn/v4.0/syncer-overview/','/docs-cn/stable/reference/tools/syncer/','/zh/tidb/stable/syncer-overview','/zh/tidb/dev/syncer-overview']
```
Comment (yikeke) [inline syncer-overview.md:3]:
```suggestion
aliases: ['/docs-cn/stable/syncer-overview/','/docs-cn/v4.0/syncer-overview/','/docs-cn/stable/reference/tools/syncer/','/zh/tidb/stable/syncer-overview']
```

**Review by yikeke — APPROVED** [review]
rest lgtm

File: syncer-overview.md (hunk 4 above)
Comment (TomShawn) [reply syncer-overview.md:3]: image
The master branch does not have this alias.

**Pull request merged**